README.md

```diff
@@ -2,7 +2,8 @@
 
 [![Build Status](https://travis-ci.org/DeveloperCAP/MLCAT.svg?branch=development)](https://travis-ci.org/DeveloperCAP/MLCAT)
 [![Maintainability](https://api.codeclimate.com/v1/badges/f1d7947c8a62338544dd/maintainability)](https://codeclimate.com/github/DeveloperCAP/MLCAT/maintainability)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/f1d7947c8a62338544dd/test_coverage)](https://codeclimate.com/github/DeveloperCAP/MLCAT/test_coverage)
+[![codecov](https://codecov.io/gh/DeveloperCAP/MLCAT/branch/master/graph/badge.svg)](https://codecov.io/gh/DeveloperCAP/MLCAT)
+[![Dependency Status](https://www.versioneye.com/user/projects/5a57a7d70fb24f382f102324/badge.svg?style=flat-square)](https://www.versioneye.com/user/projects/5a57a7d70fb24f382f102324)
 [![Requirements Status](https://requires.io/github/DeveloperCAP/MLCAT/requirements.svg?branch=master)](https://requires.io/github/DeveloperCAP/MLCAT/requirements/?branch=master)
```

Comment:
@AakankshaSanctis please remove the requires.io badge. We have a versioneye badge which is serving the same purpose.